### PR TITLE
Adding test to ignore unhandled message response for possessive refer…

### PR DESCRIPTION
**Pull request by jottinger**
…ence to bot

### CODE REVIEW

File: src/main/kotlin/javabot/Javabot.kt [hunk 1]
```diff
@@ -322,7 +322,7 @@ open class Javabot @Inject
 
         }
 
-        if (responses.isEmpty()) {
+        if (responses.isEmpty() && (!message.value.toLowerCase().startsWith("${nick}'s".toLowerCase()))) {
```
Comment (evanchooly) [inline src/main/kotlin/javabot/Javabot.kt]: should we also filter out `javabot is`  ?

**Comment by jottinger** [comment]
No. There are a lot of cases where "nick is this the right answer" etc is
incorrect grammar but something that should be responded to.

File: src/test/kotlin/javabot/operations/IgnorePossessiveAddressTest.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import javabot.MockIrcAdapter
+import org.testng.Assert.assertEquals
+import org.testng.Assert.assertNotNull
+import org.testng.annotations.Test
+
+class IgnorePossessiveAddressTest
+@Inject constructor(val ircAdapter: MockIrcAdapter, val javabot: Javabot) : BaseTest() {
+    @Test
+    fun testConfig() {
+        assertNotNull(ircAdapter)
```
Comment (evanchooly) [inline src/test/kotlin/javabot/operations/IgnorePossessiveAddressTest.kt]: This test is redundant since ircAdapter is nonnullable.

**Comment by jottinger** [comment]
Agreed, was an artifact of the development.

**Pull request merged**